readthedocs/search/tests/data/generic/in/basic.html

```diff
@@ -6,5 +6,14 @@
   </head>
   <body>
     Content of the body.
+    <dl>
+      <dt id="love">Love</dt>
+      <dd>To code is to love</dd>
+      <dt>No section for me</dt>
+      <dd>This term does not have an ID so it's not gonna get its own section</dd>
+      <dt id="code">Code</dt>
+      <dt id="docs">Docs</dt>
+      <dd>Code and docs are like love, they are 4-letter words.</dd>
+    </dl>
```

Comment:
Can we add a nested list here? I love how short this is, it's easy to see if the code is really working.

Reply:
Absolutely! I don't know if there was a test from before that showed the behavior, but all tests are updated, you can check if they are how they should be. I think stuff works like BFS.

Reply:
We should also add nested elements that don't have an ID to test with